stable/moodle/README.md

```diff
@@ -92,9 +104,55 @@ $ helm install --name my-release -f values.yaml stable/moodle
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+### Ingress without TLS
+For using ingress (example without tls):
+```console
+$ helm install --name my-release \
+  --set ingress.enabled=True,ingress.hosts[0]=moodle.domain.com,moodleUsername=admin,moodlePassword=password,mariadb.mariadbRootPassword=secretpassword,serviceType=ClusterIP stable/moodle
```

Comment:
It would require also the serviceType=ClusterIP .
As for the rest of params, I just wanted to ensure the result will be compatible with the other examples, only difference being the added ingress.
If you think otherwise it can changed of course.